### ISSUE TITLE
UI Responsiveness

### ISSUE THREAD

**Issue by tfrancart** [sparna-git/Sparnatural#430]
Is it even possible ?

**Comment by SteinerPascal** [comment]
What do you mean by responsivness? for smaller screens/devices?

**Comment by tfrancart** [comment]
Yes - this is a need that was expressed, but I can't give any details myself and this requires obviously more analysis
